src/block-components/alignment/index.php

```diff
@@ -9,51 +9,29 @@
 	exit;
 }
 
-if ( ! function_exists( 'stackable_load_alignment_frontend_polyfill_script' ) ) {
-	function stackable_load_alignment_frontend_polyfill_script() {
-
-		$user_agent = ! empty( $_SERVER['HTTP_USER_AGENT'] ) ? $_SERVER['HTTP_USER_AGENT'] : '';
-
-		if ( ! $user_agent ) {
-			return;
+$user_agent = ! empty( $_SERVER['HTTP_USER_AGENT'] ) ? $_SERVER['HTTP_USER_AGENT'] : '';
+if ( ! empty( $user_agent ) && stripos( $user_agent, 'Firefox/' ) !== false ) {
+	add_filter( 'render_block', function( $block_content, $block ) {
+		$tag = new WP_HTML_Tag_Processor( $block_content );
```

Comment:
You'll have to check for the existence of `WP_HTML_Tag_Processor` because it was just recently introduced in WP 6.2

```suggestion
		if ( ! class_exists( 'WP_HTML_Tag_Processor' ) ) {
			return $block_content;
		}
		$tag = new WP_HTML_Tag_Processor( $block_content );
```